grails-app/conf/application.yml

```diff
@@ -60,6 +60,14 @@ quartz:
         showTriggerNames: true
         showCountdown: true
         showTickingCloud: true
+    scheduler:
+        instanceId: application.yml
+        instanceName: DefaultQuartzPool
+        skipUpdateCheck: true
+    threadPool:
+        threadCount: 2
```

Comment:
We want to just make sure that this thread count is high enough for a default system. When we start up the system, there are a number of scheduled tasks that startup immediately. At the same time, these job instances check whether the database migrations are being run. I want to make sure that there are enough threads in each of the different pools to not cause weird blocking during startup.